lightning/src/ln/channel.rs

```diff
@@ -281,6 +281,13 @@ impl HTLCCandidate {
 	}
 }
 
+#[derive(Clone)]
+pub struct CounterpartyForwardingInfo {
```

Comment:
Hmm, so this isn't *actually* public because `channel` is a `pub(crate)` module (hence why its not a compile failure that there are no docs).

Reply:
I guess I'm confused how this worked on the sample? Can you access the fields?

Reply:
Oh that's weird. Yeah, I can access the fields in the sample.

Reply:
Yea, I think we should either re-export it or just move it to `ChannelManager`, right now you can't create one, which maybe isnt that important but seems strange. Moving it would also enforce our compile-time checks for missing docs, which would be good :)